content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -29,6 +29,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 - The **Add User To Chat** method ([Web](/messaging/agent-chat-api/v3.4/#add-user-to-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#add-user-to-chat)) always requires an active thread. The `require_active_thread` flag was removed.
 
+### Chat access
+- The **Transfer Chat** method ([Web](/messaging/agent-chat-api/v3.4/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#transfer-chat)) always requires an active thread.
```

Comment:
```suggestion
- The **Transfer Chat** method ([Web](/messaging/agent-chat-api/v3.4/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#transfer-chat)) now requires an active thread.
```

Reply:
It seems like in this context, "always" is already expressed by the word "require". Otherwise, it would be an optional param, which it is not.